doc/zmq_getsockopt.txt

```diff
@@ -892,6 +892,18 @@ Default value:: -1
 Applicable socket types:: all, when using VMCI transport
 
 
+ZMQ_MULTICAST_LOOP: Retrieve multicast local loopback configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Retrieve the current multicast loopback configuration. A value of `1`
+means that the multicast packets sent on this socket will be looped
+back to local listening interface.
+
+[horizontal]
+Option value type:: int
+Option value unit:: 0, 1
+Default value:: 1
+Applicable socket types:: ZMQ_RADIO
```

Comment:
perhaps ```ZMQ_RADIO, when using UDP transport``` ? Otherwise it might get confusing with pgm/norm

Reply:
Yeah I wasn't sure about that, there's also the matter of raw sockets which I haven't quite understood yet.

Reply:
The raw socket, for UDP, it's about the ZMQ_DGRAM type - it's really undocumented, but IIRC it's the equivalent of ZMQ_STREAM and it just translates zmq messages into raw datagrams (or TCP stream) without any ZMTP header.